spec/aruba/api/core_spec.rb

```diff
@@ -80,7 +80,8 @@
       end
 
       it 'does not touch other environment variables in the passed block' do
-        keys = ENV.keys - ['PWD', 'OLDPWD']
+        # FIXME: Shouldn't have to exclude JRUBY_OPTS and JAVA_OPTS
```

Comment:
Can we try copy something like what Selenium does for other committers benefit, it also pops up as a flag in the IntelliJ IDEs

Something like `#TODO: mvz - Shouldn't have to....` or `#FIXME: TODO - Mathis - Shouldn't have to...`

Reply:
I've instead create a fix for the problem in #610.